pages/tutorials/supabase-search.mdx

```diff
@@ -0,0 +1,402 @@
+---
+title: Supabase Search Integration
+description:
+  The Supabase Search Integration in BuildShip allows users to leverage Supabase for their searching and indexing needs.
+---
+
+import { Callout } from 'nextra/components';
+import Image from 'next/image';
+import supabase6 from '/public/assets/supabase6.png';
+import supabase7 from '/public/assets/supabase7.png';
+import supabase8 from '/public/assets/supabase8.png';
+import supabase9 from '/public/assets/supabase9.png';
+import supabase10 from '/public/assets/supabase10.png';
+import supabase11 from '/public/assets/supabase11.png';
+
+# Supabase Search Integration
+
+The Supabase Search Integration in BuildShip empowers users to harness Supabase for their search and indexing
+requirements through a streamlined, low-code interface. Supabase provides efficient full-text search, semantic search,
+and hybrid search capabilities to create highly relevant search experiences within your applications and website.
+
+<br />
+
+<Image src={supabase11} alt='Meilisearch' width={1200} />
+
+Leveraging BuildShip's Supabase Nodes, users can seamlessly connect to their Supabase instances, unlocking the potential
+to craft **APIs, Scheduled Jobs, and various backend workflows centered around search and indexing tasks**. This
+integration is specially crafted to reduce the need for extensive coding, facilitating straightforward management of
+search indices, documents, and configurations within the intuitive BuildShip environment.
+
+The Supabase Nodes provide a no-code interface for engaging with Supabase, ensuring a hassle-free integration of
+advanced search features into your projects. From indexing new documents and tweaking search settings to conducting
+intricate search queries, Supabase Nodes are designed to streamline these processes in a user-friendly manner.
+
+## Prerequisites ✅
+
+Before you can start using the Supabase Nodes in BuildShip, ensure that you have the following prerequisites in place:
+
+### Supabase Project
+
+You'll need access to a Supabase project. Follow these steps to prepare your account and initiate your first project:
+
+1. Navigate to [Supabase](https://supabase.com/) to either sign up for a new account or log in if you are an existing
+   user.
+2. Create a new project and configure your database as needed.
+
+It's important to note, generating API keys for secure interaction with your Supabase instance is an important step that
+follows the establishment of your project.
+
+### API Keys
+
+Supabase utilizes API keys to authenticate, granting complete operational access. For enhanced security, it is
+recommended to [generate new API keys](https://supabase.com/docs/guides/api) with restricted scopes tailored to specific
+use cases.
+
+### Host
```

Comment:
Any specific reason why we're calling the API URL as "Host"? Maybe it's better to keep this as API URL since all the nodes have that as a consistent label, what do you think?